index.html

```diff
@@ -6,17 +6,18 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./css/main.css" />
     <link rel="icon" href="./favicon.ico" />
-    <title>Aziz Mejri - Portfolio</title>
+    <title>Ahmet Bozacı - Portfolio</title>
+    <script src="script.js"></script>
   </head>
   <body>
     <header>
-      <h1 id="logo">Aziz Mejri</h1>
-      <button type="button" class="menu-bars">
+      <h1 id="logo">Ahmet Bozacı</h1>
+      <button type="button" class="menu-bars" onclick="openMenu()" >
         <i class="fas fa-bars"></i>
       </button>
       <nav id="desktop-nav">
```

Comment:
- [ ] When a user click (or tap) the hamburger button, the mobile menu should appear. Currently, it is just moving the hamburger button to the left.
![image](https://user-images.githubusercontent.com/64360974/123660212-211b3280-d83c-11eb-9667-2cb67750f55a.png)

- Kindy work on this and it's best if you check other functionalities are met. For example: - When a user clicks on the buttons inside the menu, the menu should disappear and also make sure the styling matches the one specified on [figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).